### ISSUE TITLE
popover should not be visible if update button is 

### ISSUE THREAD

**Issue by marieflorescontact** [LuatixHQ/opencti#8704]
## Environment

1. OS (where OpenCTI server runs): { e.g. Mac OS 10, Windows 10, Ubuntu 16.4, etc. }
2. OpenCTI version: { e.g. OpenCTI 1.0.2 }
3. OpenCTI client: { e.g. frontend or python }
4. Other environment details:

## Reproducible Steps

Steps to create the smallest reproducible scenario:

1.Go to an Entity on `prerealese env`  (no feature flag), e.g. an Incident Response

## Expected Output

`Update` Button

## Actual Output

`Update` button AND `Popover`

![{36FCCE94-8719-4E5D-96C8-7C3DEAF7DCDD}](https://github.com/user-attachments/assets/bfb36011-0123-48ad-b757-577eab8dc457)

**Comment by Bonsai8863** [comment]
I don't believe that all the popover removal PRs have been made and merged yet. This is a part of the overarching FAB_REPLACEMENT feature flag, hence why you're seeing this on the `prerelease env`. Once all those popover removal PRs are finished and merged, this will no longer be an issue.

We've only submitted three PRs as of the writing of this comment: for analyses, locations, and cases. Only the locations PR has been merged so far, so you'll see the expected behavior on any location. We plan on submitting more PRs for the remaining entity types soon.

**Comment by SarahBocognano** [comment]
I'm closing this issue because it was fixed with this https://github.com/OpenCTI-Platform/opencti/pull/11845
